### PR TITLE
docs: release: draft release for v1.0.0

### PR DESCRIPTION
**Pull request by ruojunm**
### Description

release: draft release for v1.0.0

### Rationale

NO changes since last release v0.2.6-hf.2, but make it as the first official version for the main-net deployment

### Example

N/A

### Changes

Notable changes: 
* N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v1.0.0
+This is the first official version for the main-net deployment
```
Comment (sysvm) [inline CHANGELOG.md:4]: Add a period at the end of the sentence.

**Review by sysvm — APPROVED** [review]
LGTM

**Comment by will-2012** [comment]
Maybe the dependent versions need to be updated?

File: CHANGELOG.md (hunk 1 above)
Comment (will-2012) [reply CHANGELOG.md:4]: Maybe the dependent versions in go.mod need to be updated?

Comment (sysvm) [reply CHANGELOG.md:4]: Waiting chain to release its tag.

**Review by sysvm — APPROVED** [review]

**Review by zjubfd — APPROVED** [review]

**Review by will-2012 — APPROVED** [review]

**Pull request merged**